docs/troubleshooting.md

```diff
@@ -1,3 +1,41 @@
 # Troubleshooting Guide
 
-Content to be added.
+## Manual cleanup
+
+During Abort/Finalize LCA performs a cleanup. This includes:
+
+- Remove the old state root
+- Cleanup precaching resources
+- Delete OADP backups CRs
+- Remove IBU files from the file system
+
+If LCA fails to perform any of the above steps it transitions to Abort/Finalize failed state.
+The condition message and log shows which steps have failed.
+User should perform manual cleanup and add `lca.openshift.io/manualCleanupDone` annotation to IBU. After observing this annotation LCA will retry the cleanup and if successful, IBU transitions to `Idle`.
+
+### Stateroot cleanup
+
+During `Abort` LCA cleanups the new stateroot and during `Finalize` LCA cleanups the old stateroot. User should inspect the log to determine why this step has failed since failure in this step indicates serious issue in node.
+You can add the `lca.openshift.io/manualCleanupDone` annotation to IBU so that LCA retries the cleanup. If manual cleanup of a stateroot is necessary follow the following steps:
+
+- Run `ostree admin status` to determine if there are any existing deployments in the stateroot. These will need to be cleaned up first. Cleanup every deployment in the stateroot using:
+
+```bash
+ostree admin undeploy <0 based index of deployment> 
+```
+
+- After cleaning up all the deployments of the stateroot, you can wipe the stateroot folder:
+
+> [!CAUTION]
+> Make sure the booted deployment is not in this stateroot.
+
+```bash
+stateroot="<stateroot to be deleted>"
+unshare -m /bin/sh -c "mount -o remount,rw /sysroot && rm -rf /sysroot/ostree/deploy/${stateroot}"
+```
+
+### OADP backup cleanup
+
+The main reason that this step can fail is connection failure between LCA and the backup backend. By restoring the connection and adding `lca.openshift.io/manualCleanupDone` annotation, LCA can successfuly cleanup backup resources.
```

Comment:
Could we add this command to guide the user on how to check the backend connectivity?
```
$ oc get backupstoragelocations.velero.io -n openshift-adp
NAME                          PHASE       LAST VALIDATED   AGE   DEFAULT
dataprotectionapplication-1   Available   33s              8d    true
```
In addition to that, in this case, it seems that user also needs to clean up all existing `deletebackuprequests.velero.io` CRs? as they were already created and had errors, OADP won't reconcile failed CRs. 
We could consider cleaning up any existing deletebackuprequest CRs maybe when initiating the cleanup maybe? so user doesn't have to manually delete these CRs on failed cleanup.

Reply:
Added the command.
I have mentioned deleting `deletebackuprequests` in the next paragraph.

We can add it, I will try it in the story https://issues.redhat.com/browse/CNF-11524

Reply:
pushed some bits related to this in #387 for you to review